samza-core/src/main/java/org/apache/samza/context/JobContextImpl.java

```diff
@@ -18,32 +18,38 @@
  */
 package org.apache.samza.context;
 
-import org.apache.samza.SamzaException;
 import org.apache.samza.config.Config;
 import org.apache.samza.config.JobConfig;
+import scala.Option;
```

Comment:
Would prefer not using scala classes from within Java code.

Reply:
This is just used as the type of a local var when calling JobConfig.getName, which returns the Option.